### DISCUSSION TITLE
Balancing power of vehicles vs terrain

### DISCUSSION THREAD

**Discussion by juffanoodle** [voxeltycoon/issues#457] [General]
**Is your feature request related to a problem? Please describe.**
Elevation changes aren't really significant to route selection. Even the first trucks, loaded, will happily drive up a hill at half their max speed, which makes having to account for geography pretty useless.

**Describe the solution you'd like**
* Make vehicles more sensitive to elevation changes - more penalty for going uphill, especially when loaded

**Comment by victorleblais** [comment]
Right now, the logic behind train physics is a bit obscure. I find the penalty for turning (for trains) to be quite severe, yet the one for elevation to be quite generous (all vehicles).

I think there's something to be done, not sure what.

I think you should not encourage players to make huge hills going up and down but instead making them avoid mountains, or spend the cash into a tunnel if they're really after efficiency.
